File: DevExpress-Demo/CS/RibbonDemo.Wpf/Modules/RibbonSimplePad/Services/BackstageViewService.cs
using DevExpress.Mvvm.UI;
using DevExpress.Xpf.Ribbon;
using System.Windows;
namespace RibbonDemo {
    public interface IBackstageViewService {
        void Close();
    }
    public class BackstageViewService : ServiceBase, IBackstageViewService {
        RibbonControl Ribbon { get { return AssociatedObject as RibbonControl; } }
        public void Close() {
            if(Ribbon != null || Ribbon.MergedParent != null) {
                Ribbon.MergedParent.CloseApplicationMenu();
            }
        }
    }
}

File: DevExpress-Demo/VB/RibbonDemo.Wpf/Modules/RibbonSimplePad/Services/BackstageViewService.vb
Imports DevExpress.Mvvm.UI
Imports DevExpress.Xpf.Ribbon

Namespace RibbonDemo

    Public Interface IBackstageViewService

        Sub Close()

    End Interface

    Public Class BackstageViewService
        Inherits ServiceBase
        Implements IBackstageViewService

        Private ReadOnly Property Ribbon As RibbonControl
            Get
                Return TryCast(AssociatedObject, RibbonControl)
            End Get
        End Property

        Public Sub Close() Implements IBackstageViewService.Close
            If Ribbon IsNot Nothing OrElse Ribbon.MergedParent IsNot Nothing Then
                Ribbon.MergedParent.CloseApplicationMenu()
            End If
        End Sub
    End Class
End Namespace
